### PR TITLE
Add behat test

### PR DESCRIPTION
**Pull request by TomoTsuyuki**
Add behat test from #3

### CODE REVIEW

File: tests/behat/notification.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  Background:
+    Given the following "courses" exist:
+      | fullname | shortname | format |
+      | Course 1 | C1        | topics |
+    And I log in as "admin"
+    And I am on "Course 1" course homepage
+    And I set the following administration settings values:
+      | notificationstatement | Notification test text |
+
+  Scenario: Course edit mode off
```
Comment (dmitriim) [inline tests/behat/notification.feature]: can you please add a couple of more scenarios to cover changing of a notification type setting?

Comment (dmitriim) [reply tests/behat/notification.feature]: Also would be good to test if notification is empty, then we won't see it on the page

Comment (dmitriim) [reply tests/behat/notification.feature]: Also would be good to test if we don't see the notification on other course pages in editing mode

Comment (TomoTsuyuki) [reply tests/behat/notification.feature]: More cases have been added for notification types and empty check.
We can see the notification on the other course pages when editing on, because it's global setting for all courses.

File: tests/behat/notification.feature [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    And I log in as "admin"
+
+  Scenario: Course edit mode on/off with info type settings
+    Given I set the following administration settings values:
+      | notificationstatement | Notification test text |
+      | notificationtype      | info                |
+    And I am on "Course 1" course homepage
+    And I turn editing mode on
+    Then I should see "Notification test text" in the "//div[contains(@class, 'alert-info')]" "xpath_element"
+    And "//div[contains(@class, 'alert-warning')]" "xpath_element" should not exist
```
Comment (tomdickman) [inline tests/behat/notification.feature]: I prefer to avoid using `xpath_element` selectors and use `css_element` selectors instead as they are a bit more ubiquitous and tie directly into DOM manipulation and the front end. (Where possible I don't even like using this, but in your case, things like the Moodle "dialogue" selector don't really work.) Could you please change these (and all other selectors) to CSS like so:
```
Then I should see "Notification test text" in the ".alert-info" "css_element"
And ".alert-warning" "css_element" should not exist
```

File: tests/behat/notification.feature [hunk 3]
```diff
@@ -0,0 +1,95 @@
+@local @local_edit_course_notification
+Feature: Edit notification
+  In order to show notification text
+  As an editor
+  I need to see a notification text when editing a course if it's set
```
Comment (tomdickman) [inline tests/behat/notification.feature]: Change to "I need to be able to set notification text and type" otherwise, grammatically, this is a bit confusing.

**Review by tomdickman — CHANGES_REQUESTED** [review]
Tested as working, just some changes for clarity and then it will be good to merge

File: tests/behat/notification.feature (hunk 2 above)
Comment (TomoTsuyuki) [reply tests/behat/notification.feature]: Thanks for you suggestion.
I replace from xpath_element to css_element.

File: tests/behat/notification.feature (hunk 3 above)
Comment (TomoTsuyuki) [reply tests/behat/notification.feature]: Updated text in Feature
